test/array.js

```diff
@@ -3,6 +3,7 @@
 const metatests = require('metatests');
 const common = require('..');
 
+// prettier-ignore
```

Comment:
Maybe move it to the entries that require specific formatting for improved readability, like `'common.splitAt'`, `'common.range'`, and `'common.sequence'`?